### PR TITLE
feat: custom locale switcher

### LINKED ISSUE #2918: Add language picker to the onboarding
Part of https://github.com/getAlby/lightning-browser-extension/issues/2917 epic

User cannot currently set their preferred language before they actually onboard to the extension
We can add small language selector in the top left corner of the screen. It would be good if it could be fixed in the corner of the screen

See [Figma designs here](https://www.figma.com/file/IvlUHnC3vkIWnQ14FppZqT/%F0%9F%8E%A8-Extension-UI-revamp?type=design&node-id=844-14307&mode=design&t=WBy4yi31pW9Wm4mb-4)

image
image

image
image
image

### PR DESCRIPTION
**Pull request by pavanjoshi914**
### Describe the changes you have made in this PR

local switcher in onboardings screen
custom design to use it diffferent places 

### Link this PR to an issue [optional]

Fixes  #2918

### Type of change

(Remove other not matching type)

- `feat`: New feature (non-breaking change which adds functionality)

### Screenshots of the changes [optional]

image

image

### Checklist

- [X] Self-review of changed code
- [X] Manual testing
- [X] Added automated tests where applicable
- [X] Update Docs & Guides
- For UI-related changes
- [X] Darkmode
- [X] Responsive layout

### CODE REVIEW

**Review by reneaaron — CHANGES_REQUESTED** [review]

File: src/app/router/Welcome/Welcome.tsx [hunk 1]
```diff
@@ -64,7 +65,10 @@ function Layout() {
   return (
     <div className="flex justify-center items-center min-h-screen">
       <div className="w-full">
-        <div className="max-w-7xl mx-auto px-4 sm:px-6 lg:px-8 mb-4">
+        <div className="absolute top-0 left-0 mt-4 ml-6 z-50">
```
[the remark this answers is not in the document]
Comment (pavanjoshi914) [reply src/app/router/Welcome/Welcome.tsx:68]: image
image

**Comment by pavanjoshi914** [comment]
true absolute positioning was a bad idea there. cause even if we scroll switcher will be visible and designs are not like that to support this. i changed layout with something else

**Comment by stackingsaunter** [comment]
Can't test without built but tACK

**Review by reneaaron — CHANGES_REQUESTED** [review]
Now the layout exceeds the window height and I see scrollbars (although there shouldn't be any).

**Comment by pavanjoshi914** [comment]
> Now the layout exceeds the window height and I see scrollbars (although there shouldn't be any).

I am unclear about the behavior we want to achieve here! it really depends on the content as well right? for eg. if i surf extension in brasil lang i can see scrollbars. as text there is longer.

locale switcher takes addditional space. even we adjust some content for en lang on other contents there will be still the scrollbars appearing

**Comment by pavanjoshi914** [comment]
https://github.com/getAlby/lightning-browser-extension/assets/55848322/b94fb3ad-f18f-441a-ba6f-ff45edd21cdf

**Comment by reneaaron** [comment]
Sorry, I should have added a screenshot. The fullscreen page now always has scrollbars since the content of the page exceeds the available height. 

image

**Review by reneaaron — APPROVED** [review]
tACK

**Pull request merged**